### ISSUE TITLE
Installer seems to be missing the step to set the 'theme' (version 4.5.1)

### ISSUE THREAD

**Issue by clubforge** [impresspages/ImpressPages#698]
I created this stackoverflow question (to which I did get an answer), however, I have tried going through the install process on two separate testing VM's, and the results were the same (upon completion of the install steps, I get the following error message):

    require(/var/web/testsite/public/pagesdemo/Theme//main.php): failed to open stream: No such file or directory in /var/web/testsite/public/pagesdemo/Ip/View.php:111
    Fatal error: require(): Failed opening required '/var/web/testsite/public/pagesdemo/Theme//main.php' (include_path='.:/usr/share/php:/usr/share/pear') in /var/web/testsite/public/pagesdemo/Ip/View.php on line 111

I was able to correct the problem after some digging, and was able to add:
    "theme" => "Air",
to the config.php file which corrected the problem.  I was also provided an alternate fix in the stack overflow as to how to set the theme in the database (untested but I assume that would work as well)

http://stackoverflow.com/questions/28663039/impresspages-4-5-1-seems-to-be-missing-the-theme-setting-after-install

This is all on a fresh install.

**Comment by maskas** [comment]
I can't reproduce this.

**Comment by clubforge** [comment]
Can you point me to the code that writes the 'theme' variable to the config file or the database? I will check my installation(s) and see if I can figure out what is going on here then, as the database appears to be populating (at least some of the configuration), as well the config.php file is being populated with proper database connection details, so I am not exactly sure why the actual 'theme' variable is not being populated.

My test environment is:

Linux: Ubuntu Server 14.04
Apache: 2.4.9 (64 bit)
MySQL: Ver 14.14 Distrib 5.6.19, for debian-linux-gnu (x86_64) using  EditLine wrapper

I have tried chown www-data.www-data to the entire file/directory structure to ensure there are no permission issues. And given the database user full admin access to the database (again so there are no permission issues)

I have tried installing from both zip download from the site (downloaded Feb 22, 2015), and also via git clone (same date).

I am comfortable accepting something is wrong with my test environment rather than with the script, however I would still like to know why this is failing (and if there is anything else that is silently failing as well).  Is there a way to enable debugging to gather more information?

**Comment by maskas** [comment]
It is in the database. Just now I see that StackOverflow has messed up the layout. This is how it was meant to look like:

Plugin: Ip
Key: theme
Value: "Air" //Quotes are important. 

And these are the values of one record in ip_storage table of the database.

**Comment by maskas** [comment]
But it would be really worth to try reinstall. Who know what other values are missing in your database.

**Comment by clubforge** [comment]
Yes I have reinstalled numerous times now on 2 different virtual machines (the above listed environment and the other VM is Ubuntu 12.10 (and default versions installed with that distro).

I am still trying to go through all the code for the installer to find where it is populating the database with the default values, trying to see why it isn't adding that key (and possibly others) as well.

**Comment by maskas** [comment]
Maybe you have corrupted files?

**Comment by clubforge** [comment]
Entirely possible, however as noted I have installed both from the zip file on the download page, as well as installed from a git checkout, both giving the same results.

**Comment by clubforge** [comment]
ok a bit of further information, installed fresh again (via git clone) and had to add the key you noted above manually (again).  Then under the Admin -> System, I noticed it said that the database was out of date?  so I clicked the update button and got this:
```html
Fatal error: Uncaught exception 'Ip\Exception\Db' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'layout' in 'ip_widget'' in /var/web/impresspages/public/impresspages/Ip/Internal/Update/Migration.php:660
Stack trace:
#0 /var/web/impresspages/public/impresspages/Ip/Internal/Update/Migration.php(660): Ip\Db-&gt;execute('ALTER TABLE `ip...')
#1 /var/web/impresspages/public/impresspages/Ip/Internal/Update/Model.php(34): Ip\Internal\Update\Migration::update_2()
#2 /var/web/impresspages/public/impresspages/Ip/Internal/Update/PublicController.php(14): Ip\Internal\Update\Model::runMigrations()
#3 [internal function]: Ip\Internal\Update\PublicController-&gt;index()
#4 /var/web/impresspages/public/impresspages/Ip/Internal/Core/Job.php(126): call_user_func_array(Array, Array)
#5 [internal function]: Ip\Internal\Core\Job::ipExecuteController_70(Array)
#6 /var/web/impresspages/public/impresspages/Ip/Internal/Dispatcher/JobDispatcher.php(39): call_user_func('\\Ip\\Internal\\Co...', Array)
#7 /var/web/impresspages/public/impres in <b>/var/web/impresspages/public/impresspages/Ip/Internal/Update/Migration.php</b> on line <b>660</b><br />
```

**Comment by maskas** [comment]
Everything you experience is a magic. Including "outdated database" message which makes it twice as magic. Do you have a real hosting to try out?

**Comment by clubforge** [comment]
Unfortunately nothing available at the moment, but I will look into getting a temporary host environment set up.  I am also going to go through the code in a lot more detail to see if I can figure out what is actually happening, I have just had an extremely busy week here so I haven't had time.  Will let you know if I can figure it out.

**Comment by Jehu** [comment]
same Error here after fresh install (git cloned and checked out tag v4.6.0).
Same solution: adding ``'theme' => "Air"`` to config.php

System:
OSX Yosemite
PHP 5.4.34
Apache 2.4.10

**Comment by maskas** [comment]
Could you debug why this particular database population step is skipped on your server?
